File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        a= []
        b= []
        c= []
        for i in range(0, len(nums)):
            if (nums[i]==0):
                a.append(nums[i])
            elif (nums[i]==1):
                b.append(nums[i])
            elif (nums[i]==2):
                c.append(nums[i])
        nums.clear()
        nums.extend(a)
        nums.extend(b)
        nums.extend(c)
        return nums

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> a;
        vector<int> b;
        vector<int> c;
        for (int i=0; i<nums.size(); i++){
            if (nums[i]==0){
                a.emplace_back(nums[i]);
            }
            else if (nums[i]==1){
                b.emplace_back(nums[i]);
            }
            else if (nums[i]==2){
                c.emplace_back(nums[i]);
            }
        }
        nums.clear();
        nums.insert(nums.end(), a.begin(), a.end());
        nums.insert(nums.end(), b.begin(), b.end());
        nums.insert(nums.end(), c.begin(), c.end());
    }
};